src/nsj_rest_lib/service/service_base.py

```diff
@@ -894,10 +898,13 @@ def _save(
 
             old_dto = None
             # Recuperando o DTO antes da gravação (apenas se for update, e houver um custom_after_update)
-            if not insert:
+            if not insert and not upsert:
```

Comment:
Se for upsert, não seria melhor tentar recuperar o antigo? Se não achar, só não faz nada...